src/Microsoft.VisualBasic/tests/ConversionsTests.cs

```diff
@@ -5,83 +5,2952 @@
+        public void ToByte_InvalidObject_ThrowsInvalidCastException(object value)
+        {
+            Assert.Throws<InvalidCastException>(() => Conversions.ToByte(value));
+        }
+
+        public static IEnumerable<object[]> ToByte_OverflowObject_TestData()
+        {
+            yield return new object[] { sbyte.MinValue };
+            yield return new object[] { (sbyte)(-1) };
+            yield return new object[] { (SByteEnum)sbyte.MinValue };
+            yield return new object[] { (SByteEnum)(-1) };
+            yield return new object[] { ushort.MaxValue };
+            yield return new object[] { (UShortEnum)ushort.MaxValue };
+            yield return new object[] { short.MinValue };
+            yield return new object[] { (short)(-1) };
+            yield return new object[] { short.MaxValue };
+            yield return new object[] { (ShortEnum)short.MinValue };
+            yield return new object[] { (ShortEnum)(-1) };
+            yield return new object[] { (ShortEnum)short.MaxValue };
+            yield return new object[] { uint.MaxValue };
+            yield return new object[] { (UIntEnum)uint.MaxValue };
+            yield return new object[] { int.MinValue };
+            yield return new object[] { -1 };
+            yield return new object[] { int.MaxValue };
+            yield return new object[] { (IntEnum)int.MinValue };
+            yield return new object[] { (IntEnum)(-1) };
+            yield return new object[] { (IntEnum)int.MaxValue };
+            yield return new object[] { ulong.MaxValue };
+            yield return new object[] { (ULongEnum)ulong.MaxValue };
+            yield return new object[] { long.MinValue };
+            yield return new object[] { (long)(-1) };
+            yield return new object[] { long.MaxValue };
+            yield return new object[] { (LongEnum)long.MinValue };
+            yield return new object[] { (LongEnum)(-1) };
+            yield return new object[] { (LongEnum)long.MaxValue };
+            yield return new object[] { float.MinValue };
+            yield return new object[] { (float)(-1) };
+            yield return new object[] { float.MaxValue };
+            yield return new object[] { float.PositiveInfinity };
+            yield return new object[] { float.NegativeInfinity };
+            yield return new object[] { float.NaN };
+            yield return new object[] { double.MinValue };
+            yield return new object[] { (double)(-1) };
+            yield return new object[] { double.MaxValue };
+            yield return new object[] { double.PositiveInfinity };
+            yield return new object[] { double.NegativeInfinity };
+            yield return new object[] { double.NaN };
+            yield return new object[] { decimal.MinValue };
+            yield return new object[] { (decimal)(-1) };
+            yield return new object[] { decimal.MaxValue };
+            yield return new object[] { "-1" };
```

Comment:
Remove this chunk of data and add
MemberData(nameof(ToByte_OverflowString_TestData)] to the test below

Reply:
Fixed